### DISCUSSION TITLE
[@uifabric/charting HorizontalBarChart]Accessibility issue about the FocusZone of Chart Title

### DISCUSSION THREAD

**Discussion by xinding1121** [microsoft/fluentui#28093] [Q&A] [answered]
Hi Community,

We have received an accessibility bug saying the chart titles of HorizontalBarChart shouldn't be focused on using the Tab key since they are non-interactive.
I see in the latest version of @uifabric/charting, the chart title is wrapped by a FocusZone component: 

https://github.com/microsoft/fluentui/blob/0f5a9008e78d8a3e89ebdecb26c48694b30d35f4/packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx#LL100-L111

![image](https://github.com/microsoft/fluentui/assets/98146379/5199abad-06d6-457a-8c2e-b98e20ecfc34)

On explorer dev tool:
![image](https://github.com/microsoft/fluentui/assets/98146379/81f1fe6f-5ff0-4795-a571-e7de6f81ab57)

Should we treat it as an accessibility bug?

Thanks!

**Comment by AtishayMsft** [comment, marked as answer]
Hey @xinding1121 We have fixed the issue of chart titles in horizontal bar chart being focusable recently. You can upgrade to the latest versions to get this fix (#27580 ).

**Reply by xinding1121** [reply]
Hi @AtishayMsft , I see this fix works for Fluent UI V8 while we are still using V7. Is there any plan to fix it for V7 also?

**Reply by AtishayMsft** [reply]
Yes, we will fix this issue for V7 as well.

**Reply by xinding1121** [reply]
Thanks! Is there an approximate time for the fix, please? I want to take note of it.

**Reply by AtishayMsft** [reply]
The fix will be available by end of June.

**Reply by AtishayMsft** [reply]
@xinding1121 This issue is now fixed in version 4.23.25 of fluent charting V7 as well.
